packages/contracts/scripts/deploy-scripts/goerli-genesis.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```

Comment:
Perhaps this should `git checkout` a particular commit

Reply:
I don't like the side effect of a script being a `git checkout` - maybe it could assert that the current git head is a specific hash

Reply:
Another option is putting the commit hash into the genesis file itself as a key value pair